packages/plugin-ext/src/plugin/plugin-manager.ts

```diff
@@ -171,7 +171,10 @@ export class PluginManagerExtImpl implements PluginManagerExt, PluginManager {
             stopFn = pluginMain[plugin.lifecycle.stopMethod];
         }
         if (typeof pluginMain[plugin.lifecycle.startMethod] === 'function') {
-            const pluginExport = pluginMain[plugin.lifecycle.startMethod].apply(getGlobal(), [pluginContext]);
+            let pluginExport = pluginMain[plugin.lifecycle.startMethod].apply(getGlobal(), [pluginContext]);
+            if (pluginExport instanceof Promise) {
```

Comment:
`instanceof` is unnecessary, you can `await` on anything

Reply:
fixed